src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -8,38 +11,57 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.view.UrlBasedViewResolver;
+import org.thymeleaf.util.StringUtils;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
 import uk.gov.companieshouse.api.model.latefilingpenalty.PayableLateFilingPenalty;
 import uk.gov.companieshouse.web.pps.controller.BaseController;
 import uk.gov.companieshouse.web.pps.exception.ServiceException;
+import uk.gov.companieshouse.web.pps.service.company.CompanyService;
 import uk.gov.companieshouse.web.pps.service.penaltypayment.PayablePenaltyService;
 import uk.gov.companieshouse.web.pps.session.SessionService;
 
 import jakarta.servlet.http.HttpServletRequest;
 import java.util.Map;
-import java.util.Optional;
 
 @Controller
-@RequestMapping("/late-filing-penalty/company/{companyNumber}/penalty/{penaltyId}/confirmation")
+@RequestMapping("/late-filing-penalty/company/{companyNumber}/penalty/{penaltyNumber}/confirmation")
```

Comment:
Might be worth looping @dgordon-ch in here to check what I'm saying here 
as per this comment - https://github.com/companieshouse/penalty-payment-web/pull/176#discussion_r1877696922

In mongo Collection("payable_resources") - NQ72384556
```
        "links" : {
            "self" : "/company/NI038379/penalties/late-filing/payable/NQ72384556",
            "payment" : "/company/NI038379/penalties/late-filing/payable/NQ72384556/payment",
            "resume_journey_uri" : "/late-filing-penalty/company/NI038379/penalty/00531369/view-penalties"
        }
```

In mongo Collection("payments") - NQ72384556
```
        "links" : {
            "journey" : "http://chs.local/payments/bzzFopQcebIzPiQ/pay",
            "resource" : "http://api.chs.local:4001/company/NI038379/penalties/late-filing/payable/NQ72384556/payment",
            "self" : "payments/bzzFopQcebIzPiQ"
        }
```

Reply:
Do we need to harmonise the naming of some of these fields to be clearer moving forward? e.g., PENALTY_REFERENCE and TRANSACTION_REFERENCE or PAYMENT_ID something like that to differentiate and clean up naming convention across apps?

Reply:
After discussion, we will have another ticket SAN-178 for harmonise the field naming, and currently will use the penaltyId for the page